extensions/resteasy-reactive/rest-client-reactive/deployment/src/main/java/io/quarkus/rest/client/reactive/deployment/RestClientReactiveProcessor.java

```diff
@@ -525,7 +550,7 @@ private ScopeInfo computeDefaultScope(Capabilities capabilities, Config config,
             if (scopeToUse == null) {
                 log.warnf("Unsupported default scope {} provided for rest client {}. Defaulting to {}",
                         scope, restClientInterface.name(), globalDefaultScope.getName());
-                scopeToUse = BuiltinScope.DEPENDENT.getInfo();
+                scopeToUse = globalDefaultScope.getInfo();
```

Comment:
As I was going over the Cork and Kerry Mountains, eh, over the code in this processor, this seemed like an obvious bug, so I thought I could just fix it.